### ISSUE TITLE
Treesitter error when running any call

### ISSUE THREAD

**Issue by franksl** [mistweaverco/kulala.nvim#40]
Hi,
I just installed this plugin as a replacement for the discontinued rest.nvim.
Whenever I try to run a call on a simple http file I get this error:
![image](https://github.com/mistweaverco/kulala.nvim/assets/484004/fcfcb843-65cc-40e1-ab9d-591ab4d753e7)

I tried reinstalling almost everything but with no success,
Thanks,
Franck

**Comment by franksl** [comment]
You're right, thanks! I skipped the requirements section in the docs, hoping to find the full procedure in the Install section, my bad.
The http call now starts but it ignores my .env file. I put the .env in the same directory as the .http file, is this correct?
Thanks,
Frank

**Comment by gorillamoe** [comment]
It is indeed correct. Can you share your config with me and obfuscate sensitive information?

**Comment by gorillamoe** [comment]
By config, I mean the http file and the .env filw

**Comment by franksl** [comment]
Sure, the .env file is as simple as this:
```
BASEURL=https://mydomain.api.com/v2
USERNAME=myusername
PASSWORD=mypwd
```

I first tried this kind of http file:
```
POST {{BASEURL}}/api/my/login
Content-Type:application/json

{
    "username": "{{USERNAME}}",
    "password": "{{PASSWORD}}"
}
```

It seems to make the call but I don't understand which values are being used, is there a way to get the full call detail?

**Comment by gorillamoe** [comment]
> is there a way to get the full call detail?

Yes, set debug = true in the options table in the kulala.setup function. 

The request will be stored in your data dir in tmp/kulala/request.txt

On my machine it points to $HOME/.local/share/tmp/kulala/request.txt

**Comment by franksl** [comment]
By looking at the request.txt it seems that only the {{USERNAME}} variable was replaced, the other ones are still displayed as placeholders,
Frank

**Comment by gorillamoe** [comment]
Will take a look when I'm back home. Will take about 3hrs I guess.

**Comment by gorillamoe** [comment]
I published `v1.6.1` and this is working for me now:

```http
POST https://httpbin.org/post HTTP/1.1
content-type: application/json
accept: application/json

{
  "username": "{{USERNAME}}",
  "password": "{{PASSWORD}}"
}
```

dotenv

```env
USERNAME=gorillamoe
PASSWORD=xxxxxxxxxxxxxxx=yyyyy1

```

**Comment by franksl** [comment]
Hi,
Thanks a lot for your support!
I tried last version but when I run I get this:
![image](https://github.com/mistweaverco/kulala.nvim/assets/484004/0366d58f-38e4-4f54-9af7-9ba3ae6705a4)

Thanks

**Comment by gorillamoe** [comment]
did you call `kulala.setup()` in your plugin manager of choice?

**Comment by franksl** [comment]
Yes, my actual Lazy config is:
```
return {
    {
        'mistweaverco/kulala.nvim',
        lazy = false,
        config = function()
            local kulala = require('kulala')
            kulala.setup({
                debug = true,
                additional_curl_options = {"-v"}
            })
            vim.keymap.set('n', "<leader>rr", kulala.run)
        end
    }
}

```

**Comment by gorillamoe** [comment]
Perfect, thanks, can you try `v1.6.2`

**Comment by franksl** [comment]
It gives me the same error... :(

**Comment by gorillamoe** [comment]
Aaaah, I think the `additional_curl_options = {"-v"}` setup option is causing this.

Can you try removing it?

**Comment by franksl** [comment]
I left setup() with no parameters but the error still perists..

**Comment by gorillamoe** [comment]
Your `:lua vim.print(require("kulala.globals").VERSION)` says `1.6.1` and you're at least on nvim `0.10.0` right?

**Comment by franksl** [comment]
The command says 1.6.2 and my version is 0.10.0, 
Thanks

**Comment by gorillamoe** [comment]
I'm running out of ideas, can you try v2.0.1 and see if it works?

**Comment by franksl** [comment]
Hi, sorry for the late response, still have the same error with 2.3.0... I'll try rewriting my tests..

**Comment by gorillamoe** [comment]
Can you try the companion tool https://github.com/mistweaverco/kulala-fmt

Maybe this helps 🤷🏾

**Comment by gorillamoe** [comment]
Any updates here?
